src/Features/Core/Portable/EditAndContinue/Remote/RemoteEditAndContinueServiceProxy.cs

```diff
@@ -130,25 +131,28 @@ private IEditAndContinueService GetLocalService()
         if (client == null)
         {
             var sessionId = await GetLocalService().StartDebuggingSessionAsync(solution, debuggerService, sourceTextProvider, captureMatchingDocuments, captureAllMatchingDocuments, reportDiagnostics, cancellationToken).ConfigureAwait(false);
-            return new RemoteDebuggingSessionProxy(solution.Services, LocalConnection.Instance, sessionId);
+            using var disposable = new ReferenceCountedDisposable<IDisposable>(LocalConnection.Instance);
```

Comment:
initial ref count will be 1.

Reply:
Need to review this one

Reply:
This one is potentially problematic, and doesn't follow the general rules for RCD. Resources can't be "partially" owned by RCD, so this should be rewritten to make it trivially clear to the reader that the value passed to the RCD instance is a new instance that was created as part of creating the RCD.